kubernetes-client/src/main/java/io/fabric8/kubernetes/client/BatchAPIGroupClient.java

```diff
@@ -38,12 +31,12 @@ public BatchAPIGroupClient(OkHttpClient httpClient, final Config config) {
   }
 
   @Override
-  public MixedOperation<Job, JobList, ScalableResource<Job>> jobs() {
-    return new JobOperationsImpl(httpClient, getConfiguration());
+  public V1BatchAPIGroupDSL v1() {
+    return adapt(V1BatchAPIGroupClient.class);
   }
 
   @Override
-  public MixedOperation<CronJob, CronJobList, Resource<CronJob>> cronjobs() {
-    return new CronJobOperationsImpl(httpClient, getConfiguration());
```

Comment:
Could we keep this as `@Deprecated+@deprecated` and point to the old `V1beta1BatchAPIGroupClient` cronjobs and jobs?

This way changes won't be that breaking.

Reply:
I don't know how you reverted these changes, but you reverted the examples too (not sure if that was on purpose).

I do think that examples should use the new non-deprecated APIs